abi/vfabia64/vfabia64.rst

```diff
@@ -352,27 +356,24 @@ Change history
    |           |               |   parameters from ``decimal`` to ``integrals``,  |
    |           |               |   and improve the description of the rules.      |
    +-----------+---------------+--------------------------------------------------+
-   |2019Q4alpha|31st October   |Github preview release with an open source        |
-   |           |2019           |license.                                          |
+   |2019Q4     |30th January   |Github preview release with an open source        |
+   |           |2020           |license.                                          |
    |           |               |                                                  |
-   |           |               |Major changes are:                                |
+   |           |               |Major changes:                                    |
    |           |               |                                                  |
    |           |               |1. New License_, with relative explanation in     |
    |           |               |   `About the license`_.                          |
    |           |               |                                                  |
    |           |               |2. New sections on Contributions_, `Trademark     |
    |           |               |   notice`_, and Copyright_.                      |
    |           |               |                                                  |
-   |           |               |Minor chanegs are:                                |
+   |           |               |Minor changes:                                    |
```

Comment:
trailing whitespaces

Reply:
it's a feature, the text is in a table :)